lib/core/common/config.dart

```diff
@@ -15,6 +15,7 @@ interface class Config {
 
   static const debugMode = kDebugMode;
   static const crashlyticsEnabled = !kIsWeb && !debugMode;
+  static const analyticsEnabled = !kIsWeb && !debugMode;
```

Comment:
what about using just one constant for both?

Reply:
are you talking about `analyticsEnabled` and `crashlyticsEnabled`?